### PR TITLE
chore(release): update release action

### PR DESCRIPTION
**Pull request by williamernest**
Update release action and config for next branch and site.

### CODE REVIEW

**Review by valentin-mladenov — APPROVED** [review]
LGTM

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -35,4 +35,4 @@ jobs:
         env:
           NETLIFY_SITE_ID: ${{secrets.NETLIFY_SITE_ID}}
           NETLIFY_AUTH_TOKEN: ${{secrets.NETLIFY_AUTH_TOKEN}}
-        run: npx netlify-cli deploy $([[ $GITHUB_REF_NAME == "main" ]] && echo "--prod" || ([[ $GITHUB_REF_NAME == "beta" ]] && echo "--prod --site v16-storybook-clarity-design" || echo "alias $GITHUB_REF_NAME"))
+        run: npx netlify-cli deploy $([[ $GITHUB_REF_NAME == "main" ]] && echo "--prod" || ([[ $GITHUB_REF_NAME == "alpha" ]] && echo "--prod --site v17-storybook-clarity-design" || echo "alias $GITHUB_REF_NAME"))
```
[the remark this answers is not in the document]
Comment (williamernest) [reply .github/workflows/release.yml:38]: My mental model was to create branches for each stage of the release cycle, but that's just trying to semantically align. I'll change this to next

Comment (williamernest) [inline .github/workflows/release.yml:38]:
```suggestion
        run: npx netlify-cli deploy $([[ $GITHUB_REF_NAME == "main" ]] && echo "--prod" || ([[ $GITHUB_REF_NAME == "next" ]] && echo "--prod --site v17-storybook-clarity-design" || echo "alias $GITHUB_REF_NAME"))
```

File: .releaserc.js [hunk 2]
```diff
@@ -5,7 +5,7 @@
  */
 
 module.exports = {
-  branches: ['main', '+([0-9]).x', { name: 'beta', prerelease: true }, { name: 'rc', prerelease: true }],
+  branches: ['main', '+([0-9]).x', { name: 'beta', prerelease: true }, { name: 'alpha', prerelease: true }],
```
Comment (williamernest) [inline .releaserc.js:8]:
```suggestion
  branches: ['main', '+([0-9]).x', { name: 'beta', prerelease: true }, { name: 'next', prerelease: true }],
```

**Review by kevinbuhmann — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (williamernest) [reply .github/workflows/release.yml]: Updated both to be next, we can just publish "next" releases as our alphas until we get to a beta.

**Pull request merged**